### DISCUSSION TITLE
Do I really need a Ramps V1.4 for my 5-Axis CNC Machine?

### DISCUSSION THREAD

**Discussion by saadbhattii** [fra589/grbl-Mega-5X#365] [Q&A] [answered]
Hello, I have been working on my 5-Axis CNC machine, I wanted to try running it on Grbl Mega 5X, my setup includes a Laptop for G-code sender, Arduino Mega 2560 (1), TB6600 Stepper motor Drivers (5), Nema 23 (5) stepper motors, 2M2880 Stepper motor Driver, and Nema 34 Stepper Motor. 
Can I directly connect my Arduino Mega 2560 to my Drivers? Or do I need a Ramps V1.4 Board? Which I honestly don't know what it does (would love an explanation), and I would also appreciate if you could explain what would be the difference if I buy and connect a Ramps board or not. 

Thankyou!

**Comment by fra589** [comment, marked as answer]
Hi @saadbhattii

The RAMPS card mainly has 2 interesting functions:
- Support for “pollolu” type drivers (for NEMA 17 motors),
- management of 12V PWM outputs useful for powering certain spindle motors...

Using the RAMPS card is not at all an obligation, it is entirely possible to directly connect your drivers and everything else directly to the Arduino Mega card.

For your configuration, the RAMPS card will do nothing and is not useful.

@++;
Gauthier.

**Reply by saadbhattii** [reply]
@fra589 Thank-you for your efforts and active engagement, you are a genius and a saint!

**Reply by fra589** [reply]
Your welcome ! :-)
